applications/dashboard/views/settings/plugins.php

```diff
@@ -18,7 +18,7 @@
     $title = '';
     $helpTitle = sprintf(t('About %s'), t('Locales'));
     $pathHelp = sprintf(
-        t('LocaleHelp', 'Locales allow you to support other languages on your site. Once a locale has been added to your %s folder, you can enable or disable it here.'),
+        t('Locales allow you to support other languages on your site.'),
```

Comment:
If we remove `Once a locale has been added to your %s folder` 
what is the reason to use `sprintf()` function?

Reply:
Good point. Please stand by for a commit.